trieste/acquisition/optimizer.py

```diff
@@ -32,6 +33,12 @@
 """ Type variable bound to :class:`~trieste.space.SearchSpace`. """
 
 
+class FailedOptimizationError(Exception):
+    """Raised when an acquisition optimizer fails to optimize"""
+
```

Comment:
Not sure you need the blank line, or even the pass (since you've got a docstring)